src/System.Runtime/tests/System/HashCodeTests.netcoreapp.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+public static class HashCodeTests
```

Comment:
There should be `Assert.Throws<NotSupportedException>(...)` tests that verify `GetHashCode` and `Equals` throw.